### PR TITLE
Draft: Azure outage Incident report

### PR DESCRIPTION
**Pull request by evolutionise**
# About

This post is an incident report from the recent Azure network outage

# Release

For release ASAP

# Blog image idea

As this a slightly more formal and time-sensitive post, we could do without one. 
If you wanted one, we could use a network style image e.g. [this one](https://www.octopus.com/blog/what-is-a-service-mesh) or some sort of warning.

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://www.octopus.design/writing/writing-tldr/).
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [x] The images I've included follow the [rules for working with images](https://www.octopus.design/writing/working-with-images/).
- [x] I've added alt text for all the images I've included (alt text describes image to people unable to see it, 125 characters max)
- [ ] One screenshot (if any are included) is suitable for Twitter. This screenshot should be 16:9 ratio and at least 1200 x 675 pixels. For screenshots from the Octopus Web Portal, please use Chrome and the browser’s zoom function to remove as much white space as possible. If screenshots are annotated (e.g., with boxes or arrows), please also provide a version that is unannotated for use on Twitter. Do not use dark mode for the Octopus Web Portal.

## Publication date
Ideally, ASAP

### CODE REVIEW

File: blog/2023-02/incident-azure-network-outage/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+| Time to incident declaration | 69 mins     |
+| Time to resolution           | 4hrs 32mins |
+
+## Incident Timeline
+
+Thursday, 25 January, 2023 _(all dates and times below are shown in UTC)_
+
+_7:45:_ An Octopus engineer notes that access to Azure’s portal is down due to request timeouts.
+Azure’s status page is still green (working) for all services. The engineer identifies that it is likely that the connectivity issues are within Azure’s internal networks regardless of the status page, as it does not always accurately reflect the current status. They also note that it is possible that the network issue is preventing Azure from updating their status page. The engineer notes that we are starting to see increased load on the database used by Octopus.com/OctopusID, and that there is a risk of customer-facing login issues if it continues.
+_7:51:_ A support engineer notes that a customer reported being unable to log in using OctopusID approximately an hour ago.
```
Comment (michaelnoonan) [inline blog/2023-02/incident-azure-network-outage/index.md:37]: These paragraphs aren't laying out how you expect. Instead, they are all inlined.

**Review by michaelnoonan — APPROVED** [review]
I approve the content. Just a formatting comment that you should change prior to publishing.

File: blog/2023-02/incident-azure-network-outage/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+title: Network Connectivity Disruption on January 25, 2023
+description: Public incident report on network connectivity disruption from January 25, 2023
+author: alix.klingenberg@octopus.com
+visibility: public
+published: 2023-02-01-1400
```
Comment (TeganAli) [inline blog/2023-02/incident-azure-network-outage/index.md:6]:
```suggestion
published: 2023-02-01
```

**Review by michaelnoonan — APPROVED** [review]
I've re-read and I approve.

**Pull request merged**